### PR TITLE
Controllers views routes 1

### PR DESCRIPTION
**Pull request by lianneg19**
Controllers, views, and routing for the tables.

### CODE REVIEW

File: Gemfile
Comment (lianneg19) [inline Gemfile:1]: What's gem 'spring'?

Comment (fnarciso2rrc) [reply Gemfile:1]: It is feature that allows the compiler to run faster.

File: db/seeds.rb [hunk 1]
```diff
@@ -65,21 +65,26 @@
...
-    add_dog = Dog.order("RANDOM()").take
-
-    if !Customer.exists?(dog_id: add_dog)
-        new_customer = Customer.create(
-            first_name: Faker::Name.first_name,
-            last_name: Faker::Name.last_name,
-            age: Faker::Number.between(from: 1, to: 100),
-            dog_id: add_dog.id
-        )
+    set_rand_dog_count = rand(0..10)
```
Comment (fnarciso2rrc) [inline db/seeds.rb]: what does this do??

Comment (lianneg19) [reply db/seeds.rb]: It generates a random number from 0-10 to be used for giving how many dogs a customer has.

**Pull request merged**